### DISCUSSION TITLE
Support for typer==0.9.0

### DISCUSSION THREAD

**Discussion by PieterBunnenberg** [tiangolo/typer-cli#134] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
❯ pip install typer==0.9.0 typer-cli==0.0.13
```

### Description

ERROR: Cannot install typer-cli==0.0.13 and typer==0.9.0 because these package versions have conflicting dependencies.

### Operating System

Linux

### Operating System Details

This field is required.

### Python Version

Python 3.12.1

### Additional Context

See:

1. https://github.com/tiangolo/typer-cli/issues/125
2. https://github.com/tiangolo/typer-cli/pull/120

**Comment by yolabingo** [comment]
Perhaps this could become an option on the main typer repo `typer[cli]`

**Comment by PieterBunnenberg** [comment]
I would also be possible to loosen the tight dependencies here: https://github.com/tiangolo/typer-cli/blob/6b88faf0740cb8e82847204e971dc615d9853aef/pyproject.toml#L43

Or add an automatic version update tool like [renovate](https://github.com/renovatebot/renovate).

Anything is welcome, but these coupled repositories must be kept in sync with each other...

**Comment by tiangolo** [comment, marked as answer]
`typer-cli` is now part of `typer` :tada: 

You no longer need to install `typer-cli`, just install `typer` and you will get the `typer` command with everything. :rocket:

This is since Typer version 0.12.1
